src/Message/Attachment.php

```diff
@@ -16,15 +16,22 @@ class Attachment extends DataObject
      * @param string $title    The attachment title.
      * @param string $text     The attachment body text.
      * @param string $fallback A plain-text summary of the attachment.
+     * @param string $callback_id A unique text callback_id.
+     * @param string $color A color value
+     * @param string $pretext Pretext value
+     * @param array $fields Attachment fields
+     * @param array $actions Attachment actions
      */
-    public function __construct($title, $text, $fallback = null, $color = null, $pretext = null, array $fields = [])
+    public function __construct($title, $text, $fallback = null, $callback_id, $color = null, $pretext = null, array $fields = [], array $actions = [])
```

Comment:
One thought here, you probably want to use `$callback_id = null` in the constructor here, otherwise it breaks the existing API contract for the Attachment object and will break code on upgrade.

If you really meant to require the field, you probably should move it before `$fallback` as this order would require all constructor calls to also specify a value, or null, for `$fallback` as well.